.travis.yml

```diff
@@ -1,7 +1,13 @@
 language: node_js
-sudo: false
 node_js:
   - '4'
-  - '5'
   - '6'
-  - '7'
+  - '8'
+  - '10'
+  - '12'
+env:
+  - ACORN_VERSION=6
+  - ACORN_VERSION=7
+
+before_script:
```

Comment:
This would be better done as override of `install` step. Then Travis won't run `npm install`, and you can just install the required peer dependency unconditionaly.

Reply:
I did it this way because doing `npm install acorn@7` on a newly cloned repo will only install acorn, not any of the package's dependencies.

Reply:
Well we don't have any other deps aside from `acorn`, so it should be fine.

Reply:
hah, that's fair enough. I might've been wrong in my last comment then, because I only tested it in my acorn-jsx clone! will update